doc/SYNDICATION.md

```diff
@@ -0,0 +1,163 @@
+
+On clicking on the `green tick` icon, the user can save or download the content
+
+![syndication_modal.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/syndication_modal.png)
+
+### Republishing Platform
+
+On the top right corner on the page, the `Republishing` link takes users to the republishing platform.
+
+![republishing_menu.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/republishing_menu.png)
+
+On the republishing tool, Users can see their contract details, allowance details and select their preferred download format.
+
+![republishing_platform.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/republishing_platform.png)
+
+Users can also manage saved, downloaded and Spanish content.
+
+## Key systems
+
+### [**n-syndication**](https://github.com/Financial-Times/n-syndication)
+
+This is the client side library responsible for adding syndication symbols next to article/podcast and video headlines on ft.com. Allowing syndication customers to download or save content for republishing.
+
+**What it does**
+
+- Checks if logged in user is a syndication user
+- Gets all the `contentIds` from the DOM
+
+![get_content_ids_snippet.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/get_content_ids_snippet.png)
+
+- Passes them to [next-syndication-api](https://github.com/Financial-Times/next-syndication-api/tree/master/doc#post-syndicationresolve) to get the syndication data for each content ID
+
+![syndication_api_response_snippet.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/syndication_api_response_snippet.png)
+
+- Uses the response to put the syndication symbols
+
+### [**next-syn-list**](https://github.com/Financial-Times/next-syn-list)
+
+This is the application for the republishing platform.
+
+The republishing platform allows users to view their contract, manage saved and downloaded content
+
+### [**next-syndication-api**](https://github.com/Financial-Times/next-syndication-api)
+
+This is the REST API responsible for powering syndication (Heart of syndication). It lives in heroku with a Postgres DB attached. It handles request from the `n-syndication` and `next-syn-list`. It also runs some cron jobs and sync tasks
+
+Here are the available [API endpoints](https://github.com/Financial-Times/next-syndication-api/tree/master/doc)
+
+**What it does**
+- Manages connection and interactions with attached Postgres DB
+- Stores most of its data in the Postgres DB
+- Every 24 hours updates information about contracts with up to date data from salesforce.
+- Data about saved and downloaded content is added to the DB
+- It stores Personal Identifiable Information (Membership API were flaky)
```

Comment:
In the future, we should investigate if this is still the case. Remove this part could make the application simpler and less critical in case of data breach.

Reply:
Yes, also different tables store user_id, name, and email which is not ideal and difficult to maintain across all of them. Would add a ticket to the backlog in regards to this.